src/index.html

```diff
@@ -11,6 +11,22 @@
   </head>
   <body>
     <header class="nav-container"></header>
+    <main class="content">
+      <h1 class="headline">Upcoming Events</h1>
+      <ul class="events_list d-grid">
+        <!-- <li class="the_event">
+          <div class="image">
+            <img src="" alt="">
+          </div>
+          <div class="event_i">
+            <h5 class="event_name"></h5>
+            <p class="event_genre"></p>
+            <p class="event_date"></p>
+            <button class="comment_btn">comment</button>
+          </div>
+        </li> -->
```

Comment:
I'm sure you used this line of code to test the structure on the browser, kindly remove all the commented lines of code before merging pls.